workflow-templates/im-run-annotate-pagerduty-template.properties.json

```
@@ -0,0 +1,5 @@
+{
+    "name": "Runbook - Annotate PagerDuty",
```

For consistency, the rest of the runbook templates start with 'Run'.
```suggestion
    "name": "Run - Annotate PagerDuty",
```